ui/lib/pki/addon/components/pki-tidy-form.hbs

```diff
@@ -0,0 +1,80 @@
+<hr class="is-marginless has-background-gray-200" />
+
+<p class="has-top-margin-m has-bottom-margin-l">Tidying cleans up the storage backend and/or CRL by removing certificates
+  that have expired and are past a certain buffer period beyond their expiration time.
+  <DocLink @path="/vault/api-docs/secret/pki#{{if (eq @tidyType 'manual') 'tidy' 'configure-automatic-tidy'}}">Learn more</DocLink>
+</p>
+
+<MessageError @errorMessage={{this.errorBanner}} class="has-top-margin-s" />
+
+<form class="has-bottom-margin-s" {{on "submit" (perform this.save)}}>
+  {{#if (eq @tidyType "auto")}}
+    {{#let this.intervalDurationAttr as |attr|}}
+      <TtlPicker
```

Comment:
Differs from designs slightly in order to follow the pattern used for `pki-configuration-edit` where related parameters are combined into a single input. In this case the ttl toggle sets `enabled` and the duration value sets `interval_duration`